WFInfo/Data.cs

```diff
@@ -119,7 +118,7 @@ private void SaveDatabase(string path, object db)
 
         public bool IsJwtAvailable()
         {
-            return JWT != null;
+            return JWT.Length > 500; //check if the token is of the right length
```

Comment:
This broke logins.

>500 means logged in
>300~ seems to mean not logged in

Current setup ignores all data traffic and does not login in correctly